.cirrus.yml

```diff
@@ -1,3 +1,18 @@
+task:
+  auto_cancellation: true
+
+  name: Black Formatting
+  container:
+      image: python3.6:latest
+
+  install_script:
+    # pip >= 19.0 crashes on Linux with Python 3.7
+    - python -m pip install --retries 3 --upgrade "pip<19.0"
+    - pip install `cat requirements/development.txt | grep black`
+
+  test_script:
+    - black --exclude vendored\|_vispy --check .
```

Comment:
could you add this `--exclude` stuff to `pyproject.toml` and remove it from here? you can find an example setup [here](https://github.com/python/black/blob/master/pyproject.toml#L12)